ios/Classes/SwiftFlutterCrispChatPlugin.swift

```diff
@@ -124,17 +124,27 @@ public class SwiftFlutterCrispChatPlugin: NSObject, FlutterPlugin, UIApplication
                 completionHandler([.alert, .sound])
             }
         } else {
-            completionHandler([])
+            // Notify Flutter or rethrow to allow other handlers to process the notification
+            channel?.invokeMethod("onNotificationReceived", arguments: notification.request.content.userInfo)
```

Comment:
Can you explain this `onNotificationReceived` channel? from where this is receiving in this package? Because I didn't create this channel.

Reply:
The `onNotificationReceived ` channel is part of the `FlutterMethodChannel` created in the `register(with:)` method of your iOS plugin, with the name `"flutter_crisp_chat"`. This method channel enables communication between the native iOS code and the Dart side of your Flutter app. When a non-Crisp notification is received or tapped, the iOS plugin uses `channel?.invokeMethod` to invoke a method (`onNotificationReceived ` in this case) on the Dart side, passing the notification data for further handling in Flutter. If you haven't explicitly implemented handling for this method on the Dart side, it must be added to process these events.

Reply:
Okay, I need to test if it is working fine then I will add this changes on next update!!